app/modules/import-export/module.php

```diff
@@ -43,6 +43,8 @@ class Module extends BaseModule {
 
 	const NO_WRITE_PERMISSIONS_KEY = 'no-write-permissions';
 
+	const NO_DOMDOCUMENT_KEY = 'no-domdocument';
```

Comment:
hmm, maybe?

```suggestion
	const NO_DOM_DOCUMENT_KEY = 'no-dom-document';
```

or if you wanna have the real reason

```suggestion
	const NO_LIBXML_EXT_KEY = 'no-libxml-ext';
```

Reply:
I liked the second one, but we do require the DOMDocument extension.